pkg/operator/ceph/cluster/watcher.go

```diff
@@ -201,47 +201,79 @@ func (c *clientCluster) fenceNode(ctx context.Context, node *corev1.Node, cluste
 	}
 	logger.Debugf("volumesInuse %s", volumesInuse)
 
-	rbdVolumesInUse := getCephVolumesInUse(cluster, volumesInuse)
-	if len(rbdVolumesInUse) == 0 {
-		logger.Debugf("no rbd volumes in use for out of service node %q", node.Name)
+	rbdVolumesInUse, cephFSVolumeInUse := getCephVolumesInUse(cluster, volumesInuse)
+	if len(rbdVolumesInUse) == 0 && len(cephFSVolumeInUse) == 0 {
+		logger.Debugf("no rbd or cephFS volumes in use for out of service node %q", node.Name)
 		return nil
 	}
 
-	logger.Infof("node %q require fencing, found rbd volumes in use", node.Name)
 	listPVs, err := c.context.Clientset.CoreV1().PersistentVolumes().List(ctx, metav1.ListOptions{})
 	if err != nil {
 		return pkgerror.Wrapf(err, "failed to list PV")
 	}
 
-	rbdPVList := listRBDPV(listPVs, cluster, rbdVolumesInUse)
-	if len(rbdPVList) == 0 {
-		logger.Debug("No rbd PVs found on the node")
-		return nil
-	}
+	if len(rbdVolumesInUse) != 0 {
+		logger.Infof("node %q require fencing, found rbd volumes in use", node.Name)
```

Comment:
Should this logging be moved down to line 222? On line 219 it might be found that no volumes actually require the fencing, right?